packages/filebrowser/src/listing.ts

```diff
@@ -1977,6 +1977,13 @@ export namespace DirListing {
         const checkboxWrapper = this.createCheckboxWrapperNode({
           alwaysVisible: true
         });
+        const checkboxInput = checkboxWrapper.querySelector(
+          'input[type="checkbox"]'
+        );
+        checkboxInput?.setAttribute(
+          'aria-label',
+          trans.__('Select all files and directories')
```

Comment:
Ideally this would be updated depending on the state ("Select all" or "Deselect all" if all are selected)

Reply:
Updated in https://github.com/jupyterlab/jupyterlab/pull/13682/commits/4b532287f77c9280eeb6d0ff22e69b30813c9dc1

Reply:
Thanks!